### DISCUSSION TITLE
windows: vipsthumbnail.exe not support unicode filename

### DISCUSSION THREAD

**Discussion by aisnote** [libvips/libvips#2593] [Q&A]
## Questions, enhancements, tips, etc.

![image](https://user-images.githubusercontent.com/1201634/115100709-3e09c000-9ef3-11eb-8483-996548ecd465.png)

Please use libvips discussions for other topics.

https://github.com/libvips/libvips/discussions

## Bug report

**Describe the bug**

**To Reproduce**
1. change file name to chinese
2. run vipsthumbnail.exe chinese.heic
3. See error
vipsthumbnail.exe: unable to thumbnail z??-?–?IMG_0021.HEIC
heifload: Input file does not exist: Unspecified: Error opening file: No such file or directory (2)

**Expected behavior**

**Actual behavior**
A clear and concise description of what actually happen.

**Screenshots**

**Environment**
(please complete the following information)
 - OS: Windows 10
 - Vips: >vipsthumbnail.exe --vips-version
libvips 8.8.0-Wed Apr 17 15:19:38 UTC 2019
 

**Additional context**

**Comment by jcupitt** [comment]
Hello @aisnote,

That's quite an old libvips. It should work with the current one -- I see:

```
$ ./vipsthumbnail.exe ~/pics/йцуке.heic
$ ./vipsheader.exe ~/pics/tn_йцуке.jpg
/home/john/pics/tn_╨╣╤å╤â╨║╨╡.jpg: 128x96 uchar, 3 bands, srgb, jpegload
$ 
```

It won't display the filename correctly on the console, but that's a separate issue.

Try:

https://github.com/libvips/libvips/releases/download/v8.10.6/vips-dev-w64-all-8.10.6.zip
